package.json

```diff
@@ -48,10 +48,10 @@
         "typescript": "^4.2.2",
         "wait-on": "^5.3.0"
     },
-    "packageManager": "pnpm@8.15.5",
+    "packageManager": "pnpm@9.13.2+sha512.88c9c3864450350e65a33587ab801acf946d7c814ed1134da4a924f6df5a2120fd36b46aab68f7cd1d413149112d53c7db3a4136624cfd00ff1846a0c6cef48a",
```

Comment:
the sha here is also a bit overkill I think version alone should be enough

Reply:
was automatically added, yeah ofc we should be able to delete it